pgmq-extension/sql/pgmq.sql

```diff
@@ -522,6 +522,76 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+CREATE FUNCTION pgmq.drop_queue(queue_name TEXT)
```

Comment:
Please also add this new function definition to a migration file - `pgmq-extension/sql/pgmq--1.44--1.5.0.sql`.